### DISCUSSION TITLE
Is <Overflow> available for vertical components?

### DISCUSSION THREAD

**Discussion by maralihart** [microsoft/fluentui#27208] [Q&A] [answered]
I'm not seeing any reference to vertical components in the documentation and am not able to make it work in my code.
Not sure if the problem is my code or if it hasn't been implemented to work vertically yet.
[Documentation](https://react.fluentui.dev/?path=/docs/components-overflow--default)

**Comment by layershifter** [comment, marked as answer]
@maralihart I created an issue to add an example to documentation, https://github.com/microsoft/fluentui/issues/27245.

**Reply by maralihart** [reply]
I was going to add an example to documentation, but found an issue: #27249

**Reply by layershifter** [reply]
@maralihart FYI #27245
